Packs/illuminate/Integrations/Analyst1/Analyst1.yml

```diff
@@ -50,11 +136,13 @@ configuration:
   - suddenDeath
   type: 17
   required: false
+  section: Collect
 - defaultvalue: '20160'
   name: feedExpirationInterval
   display: ''
```

Comment:
Please provide a descriptive `display` name for the `feedExpirationInterval` parameter. According to XSOAR guidelines, all configuration parameters should have a user-friendly display name.